examples/2016-10-31/encryption_proxy/src/decryption/index.py

```diff
@@ -0,0 +1,53 @@
+import boto3
```

Comment:
Can you move this to encryption_proxy/decryption.py? As much as we love directory structure, we'd like to keep things simple.